src/languages/en.ts

```diff
@@ -3181,7 +3181,8 @@ export default {
         overLimitAttendee: ({formattedLimit}: ViolationsOverLimitParams) => `Amount over ${formattedLimit}/person limit`,
         perDayLimit: ({formattedLimit}: ViolationsPerDayLimitParams) => `Amount over daily ${formattedLimit}/person category limit`,
         receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
-        receiptRequired: (params: ViolationsReceiptRequiredParams) => `Receipt required${params ? ` over ${params.formattedLimit}${params.category ? ' category limit' : ''}` : ''}`,
+        receiptRequired: ({formattedLimit, category}: ViolationsReceiptRequiredParams) =>
+            `Receipt required${formattedLimit ?? category ? ` over${formattedLimit ? ` ${formattedLimit}` : ''}${category ? ` ${category} category limit` : ''}` : ''}`,
```

Comment:
I think logically OR operator `||` will be better than nullish check `??` in the condition `formattedLimit ?? category` because it will check for all falsy values not only nullish values null/undefined, this can make the word `"over"` display in the message and nothing after it, because we use falsy check after `"over"` word
what do you think?

Reply:
I agree .. i just didn't use it because ts was complaining .. i have disabled this rule for this line